content/docs/pipelines/sdk/output-viewer.md

```diff
@@ -186,10 +193,17 @@ metadata fields for each type.
 - `schema`
 - `source`
 
+**Optional metadata fields:**
+
+- `storage`
+
 The `confusion_matrix` viewer plots a confusion matrix visualization of the data
 from the given `source` path, using the `schema` to parse the data. The `labels`
 provide the names of the classes to be plotted on the x and y axes.
 
+You can optionally specify `'storage': 'inline'` to embed raw content of cm_file
```

Comment:
Does `cm_file` refer the the confusion matrix file or is it a known term? If this refers to the confusion matrix as a file, I would recommend replacing `cm_file` with:

the confusion matrix file

Also in the code example below,  consider replacing cm_file with:

[CONFUSION_MATRIX_FILE]

Reply:
Updated, I can see why that's confusing.

Reply:
I changed to <CONFUSION_MATRIX_CSV_FILE>, because [] means an array in python. It's better to use <> as that's clearer, that's something user needs to change.